### PR TITLE
Feature/task

### PR DESCRIPTION
**Pull request by Vanshika-8**

### CODE REVIEW

File: todo.js [hunk 1]
```diff
@@ -5,6 +5,19 @@ const todoInput = document.querySelector('.todo-input')
 const todoItemsList = document.querySelector('.todo-items')
 
 
+const totalTasks = document.querySelector(".total-tasks span")
+const completedTasks = document.querySelector(".completed-tasks span")
+const remainingTasks = document.querySelector(".remaining-tasks span")
+ 
+function countTasks() {
+  console.log(todos.length,'TASK')
```
Comment (claudialphonse) [inline todo.js]: Remove the console logs before you push your code

File: todo.js [hunk 2]
```diff
@@ -5,6 +5,19 @@ const todoInput = document.querySelector('.todo-input')
...
+  // totalTasks.textContent = todos.length
```
Comment (claudialphonse) [inline todo.js]: remove the commented/unwanted code if not required

File: todo.js [hunk 3]
```diff
@@ -5,6 +5,19 @@ const todoInput = document.querySelector('.todo-input')
...
+  const completedTasksArray = todos.filter((item) => item.completed)
+  // console.log(completedTasksArray,'Show')
```
Comment (claudialphonse) [inline todo.js]: remove commented/unwanted code
A comment should only be used to provide some info if pushed example
```//function depicting deletion of items```

Comment (Vanshika-8) [reply todo.js]: yes i will remove it

File: todo.js [hunk 4]
```diff
@@ -42,14 +55,17 @@ function renderTodos(todos) {
       <button class="delete-button">X</button>
```
Comment (claudialphonse) [inline todo.js:57]: On line 37 where you are checking if item is completed no need of checking for item.completed=== true
```
if(item.completed)
```
will work 
read up on truthy and falsy values in Javascript

**Pull request merged**